### DISCUSSION TITLE
Drivers and software question

### DISCUSSION THREAD

**Discussion by mckinley94c** [tiny-pilot/tinypilot#1847] [General]
Does this project trigger anti bot or anti cheat for video games. Some anti cheats are "always on" so I'd rather not do a full purge of tiny pilot before I sat down to play otherwise risk a ban. I'm not familiar enough with hardware level programming please excuse my lack of experience.

**Comment by db39** [comment]
Hi @mckinley94c, thanks for your question!

>Does this project trigger anti bot or anti cheat for video games

Honestly, I don't know. I haven't read any reports from anyone saying TinyPilot triggers any anti-cheat features in games. I don't have much experience with anti-cheat features, but it would depend on what the anti-cheat software is looking for. 

As I understand it, anti-cheat typically looks for scripts, third-party tools / exploits, file modifications, and odd gameplay patterns. 

TinyPilot just acts as a USB hub with a connected mouse, keyboard, and USB drive (when using Virtual Media on TinyPilot Pro). TinyPilot doesn't install any drivers or other software on to your target machine. So if you unplug TinyPilot's USB cable and HDMI cable from your target machine, TinyPilot won't be visible to the target machine, so that may be the solution you're looking for. Our [FAQ outlining how TinyPilot appears to a target machine](https://tinypilotkvm.com/faq/target-detect-tinypilot) may also help here.

I hope that helps! Please let me know if you have any questions.
